package.json

```diff
@@ -49,7 +49,7 @@
     "@types/extend": "^3.0.1",
     "@types/lodash.isempty": "^4.4.6",
     "@types/mime-types": "^2.1.1",
-    "@typescript-eslint/eslint-plugin": "^4.24.0",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.24.0",
```

Comment:
Just wondering if we should keep the parser dependency at the same version as the eslint-plugin dependency?

Reply:
Good call - we probably should. I'll push that change as well